package.json

```diff
@@ -31,7 +31,7 @@
         "@types/mocha": "^10.0.6",
         "@types/node": "^20.10.6",
         "@vercel/ncc": "^0.38.1",
-        "chai": "^4.3.10",
+        "chai": "^5.0.0",
```

Comment:
there might be some braking changes in chai, please see https://github.com/chaijs/chai/issues/781
needs to be tested if everything still works after migrating to version 5